### PR TITLE
Publiccode.net codebases css

### PR DESCRIPTION
**Pull request by angelplasma**
Adding 2 CSS files for new Codebases pages

- **lightbox.css** - New functionality. Lightbox using pure CSS (no js dependencies). Instantiated for new Codebases pages.
- **style_codebases.scss** - New style code. Jekyll-compilable CSS specific to new Codebases pages.

This is separate from the content for the new Codebases pages (#111).

-----
[View rendered codebases/index.md](https://github.com/publiccodenet/publiccode.net/blob/publiccode.net-codebases-css/codebases/index.md)
[View rendered codebases/openzaak.md](https://github.com/publiccodenet/publiccode.net/blob/publiccode.net-codebases-css/codebases/openzaak.md)
[View rendered codebases/signalen.md](https://github.com/publiccodenet/publiccode.net/blob/publiccode.net-codebases-css/codebases/signalen.md)

### CODE REVIEW

File: codebases/codebase_openzaak.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
+<html>
```
Comment (ElenaFdR) [inline codebases/codebase_openzaak.html:1]: @angelplasma Should this file be in this PR or only in #111?

File: codebases/codebase_signalen.html [hunk 2]
```diff
@@ -0,0 +1,147 @@
+<html>
```
Comment (ElenaFdR) [inline codebases/codebase_signalen.html:1]: @angelplasma Should this file be in this PR or only in #111?

File: codebases/codebase_openzaak.html (hunk 1 above)
Comment (ElenaFdR) [reply codebases/codebase_openzaak.html:1]: Ditto for all related image files.

File: codebases/codebase_signalen.html (hunk 2 above)
Comment (ElenaFdR) [reply codebases/codebase_signalen.html:1]: Ditto for all related image files.

File: codebases/index.html [hunk 3]
```diff
@@ -0,0 +1,105 @@
+<html>
```
Comment (ElenaFdR) [inline codebases/index.html:1]: Should this file be in this PR or only in #111?

**Comment by ElenaFdR** [comment]
Travis has found a broken link to /codebases/ in _site, which I don't know how to fix. The link looks correct. Help?

**Comment by angelplasma** [comment]
In reference to this error: https://travis-ci.com/github/publiccodenet/publiccode.net/builds/198718071

It's looking for "../assets/style_codebases.css", which won't exist until the Sass CSS (.scss) file has been transpiled by Jekyll into the needed .css file.

**Review by ericherman — APPROVED** [review]
I think this is fine as-is, and more can be done to tighten it up.

I might add a comment license on font (quick search suggests that Muli is OFL (SIL Open Font License)).

**Pull request merged**